easybuild/easyblocks/p/pytorch.py

```diff
@@ -253,7 +255,31 @@ def test_step(self):
             'python': self.python_cmd,
             'excluded_tests': ' '.join(excluded_tests)
         })
-        super(EB_PyTorch, self).test_step()
+
+        (out, ec) = super(EB_PyTorch, self).test_step(return_output_ec=True)
+        if ec:
+            print_warning("Test command had non-zero exit code (%s)!" % ec)
+
+        ran_tests_regex = re.compile(r"^Ran (?P<test_cnt>[0-9]+) tests in", re.M)
+        ran_tests_hits = ran_tests_regex.findall(out)
+        test_cnt = 0
+        for hit in ran_tests_hits:
+            test_cnt += int(hit)
+
+        failed_test_regex = re.compile(r"^(?P<failed_test_name>.*) failed!\s*$", re.M)
+        failed_tests = nub(failed_test_regex.findall(out))
+        failed_test_cnt = len(failed_tests)
+
+        if failed_test_cnt:
+            test_or_tests = 'tests' if failed_test_cnt > 1 else 'test'
+            failed_tests_txt = '\n'.join('* %s' % t for t in sorted(failed_tests))
+            msg = "%d %s (out of %d) failed:\n%s"
+            print_warning(msg, failed_test_cnt, test_or_tests, test_cnt, failed_tests_txt)
+
+            max_failed_tests = self.cfg['max_failed_tests']
+            if failed_test_cnt > max_failed_tests:
```

Comment:
An adjusted error for `max_failed_test==0` would be good.
Suggestion for `max_failed_test==0`: raise the above warning as an error instead

Reply:
fixed in 82011bf